akka-stream/src/main/scala/akka/stream/Attributes.scala

```diff
@@ -365,10 +364,34 @@ object Attributes {
 object ActorAttributes {
   import Attributes._
   final case class Dispatcher(dispatcher: String) extends MandatoryAttribute
+
+  object Dispatcher {
```

Comment:
@patriknw couldn't we just make this object `private[akka]`?

Reply:
wouldn't that change the existing companion (from the `case class`) and therefore not be compatible?